packages/template-retail-react-app/app/pages/cart/index.jsx

```diff
@@ -86,86 +134,99 @@ const Cart = () => {
             showError()
         }
     }
+    /**************** Wishlist ****************/
 
-    useEffect(() => {
-        // Set the default shipping method if none is already selected
-        if (basket.basketId && basket.shipments.length > 0 && !basket.shipments[0].shippingMethod) {
-            ;(async () => {
-                const shippingMethods = await basket.getShippingMethods()
-                basket.setShippingMethod(shippingMethods.defaultShippingMethodId)
-            })()
-        }
-    }, [basket.basketId])
-
-    if (!basket?.basketId) {
-        return <CartSkeleton />
-    }
-
-    if (!basket?.productItems) {
-        return <EmptyCart isRegistered={customer.isRegistered} />
-    }
-
+    /***************************** Update Cart **************************/
     const handleUpdateCart = async (variant, quantity) => {
+        console.log('variant, quantity', variant, quantity)
+
         // close the modal before handle the change
         onClose()
+        // using try-catch is better than using onError callback since we have many mutation calls logic here
```

Comment:
Good to explain this in a PR, but better as a PR comment instead of a code comment